### DISCUSSION TITLE
La Pokedex a FULL. Gráficas, botones de navegación Todo!!!

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#532] [Show and tell]
Damas y caballeros, les presento mi pokedex.
Me clave en este proyecto como no tienen idea
Pueden buscar por nombre o número, de ahí pueden explorar las diferentes evoluciones y formas de cada poke o simplemente recorrer la lista uno por uno
Aprendí a hacer vectorización en photoshop para traer a Rotom a la vida
[Aquí el Link para que la prueben](https://carlos-gmx3.github.io/Practicas4-JS.Frontend/)
Para el input solo tienen que pasar el mouse por encima del header
Los ojos cambian de color de un estado normal, buscando datos, coincidencia encontrada e incluso error
Espero les guste!!!
![rotomDex](https://user-images.githubusercontent.com/99064239/160231471-5c3a886e-5c5c-4f9c-80dd-1cdfaee3250a.png)

![cpokemon com_mapa_rotomdex](https://user-images.githubusercontent.com/99064239/160231546-995d88f9-bc3f-4cf4-8de6-afc8a3babdd2.jpg)
![Rotom Launch-x023422](https://user-images.githubusercontent.com/99064239/160231639-5acf1842-a327-44ac-973a-fd4d0e513992.png)

**Comment by JuanSalvador73** [comment]
¡Guah! está increíble, oye, danos un tutorial para hacer algo así, ¿no? ya quiero ver como haces el de VUE. Felicidades, está de lujo este :D

**Reply by carlos-gmx3** [reply]
Fue una de consultas que pa' que te cuento
Partes del poke, luego la especie, de ahí esta la linea evolutiva, las variantes los tipos...
y creo q al header le dedique un día completo con un uso super básico de photshop para crear formas, pensé en usar la misma imágen pero se pixeleaba xD

**Comment by ErickMontesDK** [comment]
Me gustó mucho la interacción con el rotom. Tambien la grafica de stats. Te felicito.

**Comment by RodolfoMorquecho** [comment]
Te quedo demasiado bien, se nota el empeño que le pusiste.

**Comment by stariluz** [comment]
Te quedo excelente amigo! De los mejores que he visto
